k6/run-all-before.sh

```
@@ -0,0 +1,89 @@
+    sleep 10
+
+    # 결과 디렉토리 생성
+    mkdir -p "$SCRIPT_DIR/results/$PHASE/$seed"
+
+    # 4. VU별 테스트 실행
+    for vu in "${VU_LEVELS[@]}"; do
+        run_count=$((run_count + 1))
+        echo ""
+        echo "--- [$run_count/$total_runs] $seed @ $vu ---"
+
+        k6 run --env LOAD=$vu \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)